lib/model/om/BaseUser.php

```diff
@@ -84,32 +84,32 @@ public function __isset($name)
     {
     }
 
-    if ('auditLogs' == $name)
+    if ('aclPermissions' == $name)
```

Comment:
(Same as previous comment.)

Reply:
Fixed.

Reply:
Based on our discussion below, I have reverted my fix to re-order these items.